apps/dapp-console-api/src/routes/auth/AuthRoute.ts

```diff
@@ -0,0 +1,22 @@
+import { Route } from '../Route'
+
+export class AuthRoute extends Route {
+  public readonly name = 'auth' as const
+
+  public readonly logoutUser = 'logoutUser' as const
+  public readonly logoutUserController = this.trpc.procedure.mutation(
+    async ({ ctx }) => {
+      const { session } = ctx
+
+      delete session.user
+
+      await session.save()
```

Comment:
we might want to try/catch this and log an error here if this call fails so we can see whats happening here